core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConfig.java

```diff
@@ -157,16 +157,10 @@ public interface TestConfig {
     @WithDefault("false")
     boolean flatClassPath();
 
-    /**
-     * The profile to use when testing the native image
-     */
-    @WithDefault("prod")
-    String nativeImageProfile();
```

Comment:
What's the reason for dropping this one?

Reply:
This the deprecated property that is being removed